package.json

```diff
@@ -31,29 +31,29 @@
         "build": "tsc -b",
         "test": "tsc -b test && node --experimental-vm-modules node_modules/jest/bin/jest.js --ci",
         "lint": "eslint .",
-        "format": "eslint src/**/*.ts test/**/*.ts examples/**/*.ts --fix",
+        "format": "prettier './(src|test|examples)/**/!(*.d).ts' --write",
```

Comment:
Let's stick to the same command for formatting and linting

Reply:
I tried the eslint version and didn't work for one file I edit on purpose, but prettier was able to format it.

Reply:
let me know if there is a simpler configuration.